iceoryx_posh/source/mepoo/mem_pool.cpp

```diff
@@ -62,7 +62,8 @@ MemPool::MemPool(const greater_or_equal<uint32_t, CHUNK_MEMORY_ALIGNMENT> chunkS
     else
     {
         IOX_LOG(FATAL) << "Chunk size must be multiple of '" << CHUNK_MEMORY_ALIGNMENT << "'! Requested size is "
-                       << chunkSize << " for " << numberOfChunks << " chunks!";
+                       << static_cast<uint32_t>(chunkSize) << " for " << static_cast<uint32_t>(numberOfChunks)
```

Comment:
I just discovered that the logger currently does not invoke implicit conversion of `greater_or_equal<T>` to `T` correctly due to SFINEA on `LogStream& operator<<(const T value)` for arithmetic typs. `LogStream& operator<<(const bool value)` does not have SFINEA so `greater_or_equal<uint32>` gets implicitly converted into bool. I guess we should replace the SFINEA implementation for arithmetic types with explicit methods.

Reply:
@elBoberido Can you please open an issue for this?

Reply:
@elfenpiff I added a task to #1755